### PR TITLE
[BPK-1203] Add backpack usage analyser

### PR DESCRIPTION
**Pull request by matthewdavidson**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+language: node_js
+node_js:
+  - lts/carbon
```
Comment (k0nserv) [inline .travis.yml]: Can Travis not pick this up from `.nvmrc`?

File: src/analyser.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+const globby = require('globby');
+
+const readFile = util.promisify(fs.readFile);
+
+const config = require('./config');
+
+const availablePlatforms = Object.keys(config);
+
+module.exports = async (platform) => {
+  if (availablePlatforms.indexOf(platform) === -1) {
```
Comment (k0nserv) [inline src/analyser.js]: __Nitpick:__ a `Set` and using `has` is nicer for this kind of thing

Comment (matthewdavidson) [reply src/analyser.js]: How about `availablePlatforms.includes` instead? If i use a `Set` then i'll have to refactor the following code:
```
console.error(`😱  Unrecognised platform, please specify one of: ${availablePlatforms.join(', ')}`);
```

Comment (k0nserv) [reply src/analyser.js]: Yeah sounds good also noticed you are getting the keys from an object here so creating a set is not great anyway

**Review by k0nserv — APPROVED** [review]
LGTM

**Pull request merged**